hapi/src/config/eos.config.js

```diff
@@ -1,9 +1,15 @@
 module.exports = {
   networkName: process.env.HAPI_EOS_API_NETWORK_NAME,
-  apiEndpoint: process.env.HAPI_EOS_API_ENDPOINT,
+  apiEndpoints: process.env.HAPI_EOS_API_ENDPOINTS
+  ? JSON.parse(process.env.HAPI_EOS_API_ENDPOINTS)
+  : [],
+  apiEndpoint: process.env.HAPI_EOS_API_ENDPOINTS
+  ? JSON.parse(process.env.HAPI_EOS_API_ENDPOINTS)[0]
+  : '',
   stateHistoryPluginEndpoint:
     process.env.HAPI_EOS_STATE_HISTORY_PLUGIN_ENDPOINT,
   chainId: process.env.HAPI_EOS_API_CHAIN_ID,
+  eosChainId: 'aca376f206b8fc25a6ed44dbdc66547c36c6c33e3a119ffbeaef943642f0e906',
```

Comment:
why is the chain ID no longer an env var?

Reply:
When a producer doesn't have a chains.json to get the bpjson, the /bp.json is used, and the nodes are deleted because it is not from the current network.
But I understand that the /bp.json is from eos, so that variable is used to compare it with the env variable. In this case are the nodes also ignored?